CONTRIBUTORS.md

```diff
@@ -1,20 +0,0 @@
-TableView contributors (sorted alphabetically)
```

Comment:
I've replaced this file with a link to GitHub's [contributors' list](https://github.com/evrencoskun/TableView/graphs/contributors), so there no need to manually maintain a file. Let me know if you prefer to keep this file.

Reply:
My only comment is that GitHub's contributors' list, only covers direct contributions and not those pulled in from forks.